server/plugins/org.eclipse.glsp.ide.editor/src/org/eclipse/glsp/ide/editor/operations/handlers/IdePasteOperationHandler.java

```diff
@@ -24,19 +24,19 @@
 import org.eclipse.glsp.graph.GModelElement;
 import org.eclipse.glsp.ide.editor.clipboard.ClipboardService;
 import org.eclipse.glsp.ide.editor.clipboard.ui.JsonTransfer;
+import org.eclipse.glsp.server.gmodel.GModelPasteOperationHandler;
 import org.eclipse.glsp.server.gson.GraphGsonConfigurationFactory;
 import org.eclipse.glsp.server.operations.OperationHandler;
 import org.eclipse.glsp.server.operations.PasteOperation;
-import org.eclipse.glsp.server.operations.gmodel.PasteOperationHandler;
 
 import com.google.inject.Inject;
 
 /**
- * An {@link OperationHandler} similar to the GLSP {@link PasteOperationHandler},
+ * An {@link OperationHandler} similar to the GLSP {@link GModelPasteOperationHandler},
  * except that it relies on the local clipboard (instead of relying on Data passed by the client
  * via the operation data).
  */
-public class IdePasteOperationHandler extends PasteOperationHandler {
+public class IdePasteOperationHandler extends GModelPasteOperationHandler {
```

Comment:
Should't we rename the extended class as well to `IdeGModelPasteOperationHandler`? And maybe move it into a a dedicated gmodel subpackage (like we did for the base GModel implementations)?

Reply:
Yes, good catch! I'll do that change alongside a rebase after merging https://github.com/eclipse-glsp/glsp-eclipse-integration/pull/52. Thanks!